stdlib/stdlib_test.go

```diff
@@ -72,6 +72,17 @@ if !is_error(cmd) {
 
 }
 
+func TestSortModule(t *testing.T) {
+	expect(t, `
+sort := import("sort")
+a := [4, 5, 3, 1, 2]
+a = sort.sort(a, func(i, j) {
+	return a[i] < a[j]
+})
+out := import("json").encode(a)
+`, []byte("[1,2,3,4,5]"))
```

Comment:
Can we add some more test coverage please?

Examples: 
less is not a function
less is an invalid function 
arr is not an array
arr is an empty array